tests/spread/integration/libasan6/task.yaml

```
@@ -0,0 +1,9 @@
+summary: Integration tests for libasan6
+
+execute: |
+  rootfs="$(install-slices libasan6_libs)"
+
+  apt install -y gcc
```

the CI is failing because you're missing a `--update` here. The test environment is a fresh new Ubuntu instance, so this is needed before installing anything else.
```suggestion
  apt install -y --update gcc
```